scripts/check_frontend_coverage.py

```diff
@@ -375,6 +374,7 @@
     'truncate.pipe.ts',
     'tutor-card.directive.ts',
     'unicode-string-editor.directive.ts',
+    'unit-test-utils.ts',
```

Comment:
So this file is going to stay here till the migration finishes right? Maybe add a comment explaining this, so someone trying to get stuff out of this list doesn't spend time trying to cover this file.

Reply:
Done!